app/views/sessions/sessioncontent.scala.html

```diff
@@ -0,0 +1,89 @@
+@(session:SessionInfo)(implicit request: Request[_], flash: Flash)
```

Comment:
Would it make more sense to use youtubeURL and slideShareURL instead of youtubeLink and slideShareLink in this template?